BasicSteps/SweepParameterStep.cs

```diff
@@ -224,11 +230,41 @@ public override void PrePlanRun()
             base.PrePlanRun();
             iteration = 0;
 
+            if (csvSweepValues != null && csvSweepValues != "")
+            {
+                // Populates sweep tables with csv from import path
+                var timer = System.Diagnostics.Stopwatch.StartNew();
+                Type plugin = PluginManager.GetPlugins<ITableImport>().First();
+                var fileHandler = (ITableImport)Activator.CreateInstance(plugin);
```

Comment:
You probably want something like `TypeData.GetDerivedTypes<ITableImport>().FirstOrDefault(td => td.CanCreateInstance)?.CreateInstance();` instead. We don't really use `PluginManager.GetPlugins` anymore.